.dockerignore

```diff
@@ -2,3 +2,5 @@
 !assembly/assembly-main/target/eclipse-che-*
 assembly/assembly-main/target/eclipse-che-*.tar.gz
 assembly/assembly-main/target/eclipse-che-*.zip
+
+!assembly/assembly-main/src/assembly/bin/launcher.sh
```

Comment:
yes, good catch on the wrong file.
we should also probably ignore all of the files in the dockerfiles/ directory as well